schema-template/items-base.json

```diff
@@ -140,17 +140,26 @@
 				"type": {
 					"$ref": "items-shared.json#/$defs/itemType"
 				},
+				"typeAlt": {
+					"$ref": "items-shared.json#/$defs/itemType"
+				},
 				"rarity": {
 					"$ref": "items-shared.json#/$defs/itemRarity"
 				},
 				"source": {"$ref": "util.json#/$defs/source"},
 				"page": {"$ref": "util.json#/$defs/page"},
+				"baseItem": {
```

Comment:
![image](https://github.com/TheGiddyLimit/5etools-utils/assets/52298102/62d543de-8408-4be7-be62-d76e242f1860)

It's just these two sources, I do agree it's a bit strange on the basis of what "baseitem" actually means on the site. It can probably be binned if you prefer.

Reply:
grand; "get rid" it is. You happy to update this + the brew fix-up PR? I should get to it all tonight, hopefully

Reply:
Yep will update all, will be after 7pm this evening however, compressed hours n that

Reply:
All done